### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by YevhenOstrovskiy**
- [DEMO LINK](https://YevhenOstrovskiy.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://YevhenOstrovskiy.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,75 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap';
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  box-sizing: border-box;
+
+  --moyo-color: #00acdc;
+
+  width: 100%;
```
Comment (dinaminahi) [inline src/style.css:10]: Let`s place it before  the variable to avoid unnecessary empty line

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,39 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="menu">
+        <ul class="menu-list">
+          <li class="menu-item">
```
Comment (dinaminahi) [inline src/index.html:18]: Check the code style rules, please
image

**Review by dinaminahi — CHANGES_REQUESTED** [review]
Good job ⭐ Just few moments to fix

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,74 @@
...
+  font-family: Roboto, sans-serif;
   margin: 0;
+  box-sizing: border-box;
+  width: 100%;
+  
+  --moyo-color: #00acdc;
+}
+
+.header {
+  box-sizing: border-box;
```
[the remark this answers is not in the document]
Comment (YevhenOstrovskiy) [reply src/style.css:13]: StyleLint error "Expected '*' to have no more than 0 universal selectors (selector-max-universal)" That rule restricts the usage of universal selectors (*) in my CSS.

**Comment by YevhenOstrovskiy** [comment]
Understand! Thank you)

вт, 28 нояб. 2023 г. в 17:11, Anastasiia Otrubchak ***@***.***

**Pull request closed without merging**